### ISSUE TITLE
Input required for 'SubTotal' / Eingabe fehlt für Wert 'Zwischensumme'

### ISSUE THREAD

**Issue by Steinblock** [buchen/portfolio#2094]
**Describe the bug**

I transfered a crypto currency (Litecoin / LTC) from an exchange to a wallet.

Because this requires a fee (0.0000042 LTC) I created a `Delivery (outbound)`

Usually this works (i.e. for BTC / ETH with higher fees) but in that case I noticed that the quote field was empty.

I thought I accidentially set the value to zero and wanted to edit the record but this is not possible anymore.

**To Reproduce**

* Create an outbound transaction

![image](https://user-images.githubusercontent.com/6850324/108332871-92810f80-71d0-11eb-8bee-58d0cc79665c.png)

* The `Quote` field should show the current price but it is zero (it is not if you have a higher amount)

![image](https://user-images.githubusercontent.com/6850324/108332932-a593df80-71d0-11eb-8cd7-83adff303992.png)

* Try to edit the entry, you can't save it because it will show a validation error: Input required for 'SubTotal' 

![image](https://user-images.githubusercontent.com/6850324/108333272-0de2c100-71d1-11eb-8a99-1935d05738ad.png)

**Expected behavior**

Even if the amount is so low that the price is basically zero (0,0009156 EUR) I should still be able to edit the record.

**Desktop:**
 - Windows 10
 - Version 0.50.4 (Feb 2021)

**Comment by metal450** [comment]
I posted a duplicate of this (allow $0 transactions), but for a different use case.  Closed my duplicate issue, but here was my description:

***

There seems to be a check in the software preventing any sale where the amount totals $0 (it shows the error "Input required for 'Total'). In the case of cryptocurrency, it would be much more convenient if this were allowed. Why? In PP, fees are always paid out in 'currency,' not in 'securities' - but in crypto, fees are paid in the crypto itself (i.e. ethereum gas). To document this in PP, it takes 2 transactions each time: a Sell transaction to convert the crypto to USD, followed by a Fees transaction for the same amount. However, if "Sell" transactions weren't prevented from having a total of $0, this could be documented as just one "Sell" where the amount equals the fees. i.e. in the Sell dialog, the amount is $10, fees is $10, thus credit total is $0. The result is the as desired: the crypto 'shares' were removed, you booked $10 in fees, and no change to your USD holdings - but it took half as many transactions.

As my transaction history grows increasingly cluttered with Sell, Fee, Sell, Fee, Sell, Fee, I think it'd be a lot more crypto-friendly to just remove this 'total can't be $0' check :)

**Comment by Kansai53** [comment]
I also have this problem, is there a reason it cant let you sell for $0? I always sell for 0.01, then put a small 'reconciliation' transaction to get rid of the extra cents every few months.  But it would be much easier if it just wouldn't force you to sell for a nonzero amount.

**Comment by chirlu** [comment]
> is there a reason it cant let you sell for $0?

Yes: The proper transaction type for a “sell” that doesn’t result in money on an account is outbound delivery.

The issue described here by @Steinblock is different.

**Comment by metal450** [comment]
> The issue described here by @Steinblock is different.

I only posted it here because I was explicitly told this was a duplicate (when I made a separate issue, https://github.com/buchen/portfolio/issues/2103)

> Yes: The proper transaction type for a “sell” that doesn’t result in money on an account is outbound delivery.

Then perhaps for UI clarity, it would be better to display that info on the dialog - i.e. "Sell with an amount of $0 should be input as Outbound Delivery," vs "Input required for sub-total."

**Comment by chirlu** [comment]
*This* issue is actually a duplicate of #2005 – see in particular https://github.com/buchen/portfolio/issues/2005#issuecomment-766863936 for an analysis.
